src/lib/swapper/swappers/ThorchainSwapper/utils/longTailHelpers.ts

```diff
@@ -96,7 +97,11 @@ export const generateV3PoolAddressesAcrossFeeRange = (
         tokenB,
         fee,
       })
-      poolAddresses.set(poolAddress as Address, fee as FeeAmount)
+      poolAddresses.set(poolAddress as Address, {
+        fee: fee as FeeAmount,
+        token0: token0.address as Address,
+        token1: token1.address as Address,
```

Comment:
Since we're doing === checks below, can we use `getAddress()` here for safety, to ensure the same chekcsum is always present when comparing addresses, avoiding compared checksummed and non-checksummed versions which would result in failing checks and potential bugs? 🙏🏽

Reply:
Excellent suggestion! https://github.com/shapeshift/web/pull/5902/commits/4bde8e1a0bb1460511b5d4254c174975d04efeb4